Snakefile

```diff
@@ -70,17 +72,30 @@ rule download:
 rule decompress:
     message: "Parsing fasta into sequences and metadata"
     input:
-        sequences = rules.download.output.sequences,
-        metadata = rules.download.output.metadata
+        sequences = "data/sequences_{serotype}.fasta.zst",
```

Comment:
As it stands, the data files produced by a local ingest must go up to S3 then be downloaded here to run the actual build. This isn't ideal -- we should be able to run a full build locally without touching S3. 

(There's the larger question of whether we should maintain multiple Snakefiles, which would largely solve this, but I'll discuss this elsewhere.)